internal/rukpak/source/image_registry.go

```diff
@@ -114,17 +115,16 @@ func (i *ImageRegistry) Unpack(ctx context.Context, bundle *BundleSource) (*Resu
 		hexVal := strings.TrimPrefix(digest.DigestStr(), "sha256:")
 		unpackPath := filepath.Join(i.BaseCachePath, bundle.Name, hexVal)
 		if stat, err := os.Stat(unpackPath); err == nil && stat.IsDir() {
-			l.V(1).Info("found image in filesystem cache", "digest", hexVal)
```

Comment:
Why are we removing logs?

Reply:
The logs would end up in the benchmarks which would eventually result in an error when benchstat is run against the benchmarks.

Reply:
@tmshort it might be possible to suppress logs if we detect benchmarks are being run. Would that be a preferred approach?

Reply:
The controller-runtime zap logger will to write to `io.Discard`  which consequently, will require no changes to the Unpack() function.